src/test/suite/providers/formatting/formattingProvider.utils.test.ts

```diff
@@ -10,32 +10,32 @@ suite('FormattingProvider utils', () => {
         let dataList = [
             // {
             //     in: , // input test string
-            //     out: , // expected result
+            //     rs: , // expected result
```

Comment:
I'm more of an `out` fan myself, I don't think this is more readable. "What's the opposite of `in`?" Answer: "`out`", not "`rs`" personally

Reply:
Nvm, nbd. I'm new to GitHub reviews, I was only looking at one commit, not all changes. This looks good 👍